controllers/akodeploymentconfig/cluster/cluster_controller_unit_test.go

```diff
@@ -149,6 +151,9 @@ func unitTestAKODeploymentYaml() {
 								},
 							},
 							DisableStaticRouteSync: pointer.BoolPtr(true),
+							FeatureGates: akoov1alpha1.FeatureGates{
+								GatewayAPI: pointer.Bool(true),
```

Comment:
This one needs to convert to a string, otherwise, it is not aligned with the ako's carvel package template

Reply:
good catch. Will fix that. Thanks!